genic/allvars.c

```diff
@@ -9,6 +9,8 @@ int ProduceGas;
 int Seed;
 int Unitary;
 int UsePeculiarVelocity;
+int PairedSim;
+int FixedAmpSim;
```

Comment:
FixedAmpSim does the same as Unitary, so you can drop it. My guess is that this is a rebase issue...